modules/db/src/main/java/org/mule/module/db/internal/config/domain/query/ParameterizedQueryTemplateFactoryBean.java

```diff
@@ -136,6 +137,14 @@ public boolean evaluate(Object object)
         return null;
     }
 
+    private Object checkNullValue(Object value)
```

Comment:
I added this method to set with a NULL value, literal "null" string.
I amn't sure, It was the expected behavior.